src/claimList.js

```diff
@@ -3,6 +3,16 @@ import {
   setupAnalytics,
   updateIcon,
   addFeedbackLink,
+  makeMobileFriendly,
+  styleBody,
+  html,
+  HEADER_HTML,
+  FOOTER_HTML,
+  RETURN_TO_TOP_LINK,
+  getClaimTypeContent,
+  getClaimStatusContent,
+  getFormattedDate,
+  getUnstyledButton,
 } from "./modules/shared.mjs";
 
 if (document.readyState === "loading") {
```

Comment:
Dust: Could this be moved into shared.mjs as its own higher order function that that can accept executeOverride as an argument?